packages/commonwealth/server/api/index.ts

```diff
@@ -1,20 +1,29 @@
 import { express, trpc } from '@hicommonwealth/adapters';
 import { Router } from 'express';
 import swaggerUi from 'swagger-ui-express';
+import { NEW_SUBSCRIPTION_API_FLAG } from '../config';
 import * as community from './community';
 import * as feed from './feed';
 import * as integrations from './integrations';
+import * as subscription from './subscription';
 import * as thread from './threads';
 
 /**
  * API v1 - tRPC Router
  */
-const apiV1 = trpc.router({
+const procedures = {
```

Comment:
should we call this procedures or features?

Reply:
Aren't these just groups of trpc procedures?

Reply:
these are technically how we "group" endpoints to model artifacts... can be called artifacts ... procedures doesn't sound right